test/integration/multikueue/suite_test.go

```diff
@@ -297,17 +306,20 @@ var _ = ginkgo.BeforeSuite(func() {
 		wg := sync.WaitGroup{}
 		wg.Add(3)
 		go func() {
+			defer ginkgo.GinkgoRecover()
```

Comment:
Also, please send in a small preparatory PR.

Reply:
#6113

Reply:
thx